lambda/templates/infrastructure/template.hbs

```diff
@@ -2,14 +2,25 @@ AWSTemplateFormatVersion: "2010-09-09"
 Transform: AWS::Serverless-2016-10-31
 Description: "Digital Identity {{ kebabCase criName }}"
 
+Parameters:
+  CodeSigningConfigArn:
+    Type: String
+    Default: ""
```

Comment:
`""` is the default absence of value, not `"none"`. The pipeline either passes the value in, in which case it's a non-empty string ot it doesn't, in which case it's an empty string. `"none"` is a bad special-case value to use, since you still need to handle the empty string so there's no need to introduce magic values and just do the empty string check instead to avoid errors